<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_create_new_game</class>
 <widget class="QDialog" name="Dialog_create_new_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>U2P Simulator - Create New Game</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/Add_Plus_Square.svg</normaloff>../Images/Add_Plus_Square.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_create_new_game">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>868</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 32px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
   </property>
   <property name="text">
    <string>Создание новой игры</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_comands_name">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>80</y>
     <width>300</width>
     <height>530</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_comands_name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>Названия команд</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_company_name_1">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>56</y>
      <width>30</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_company_name_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>115</y>
      <width>30</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_company_name_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>174</y>
      <width>30</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>3.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_company_name_4">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>233</y>
      <width>30</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>4.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_company_name_5">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>292</y>
      <width>30</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>5.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_company_name_6">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>351</y>
      <width>30</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>6.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_company_name_7">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>410</y>
      <width>30</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>7.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_company_name_8">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>469</y>
      <width>30</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>8.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name_1">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>56</y>
      <width>238</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
border: 2px solid #1E1E1E;
padding-top: -1px;

color: #1E1E1E;
text-align: center;
font-feature-settings: 'clig' off, 'liga' off;
font-family: Montserrat ExtraBold;
font-size: 16px;
font-style: normal;
font-weight: 800;
line-height: 24px;

padding-top: -1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Обязательное поле</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name_2">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>115</y>
      <width>238</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
border: 2px solid #1E1E1E;
padding-top: -1px;

color: #1E1E1E;
text-align: center;
font-feature-settings: 'clig' off, 'liga' off;
font-family: Montserrat ExtraBold;
font-size: 16px;
font-style: normal;
font-weight: 800;
line-height: 24px;

padding-top: -1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Обязательное поле</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name_3">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>174</y>
      <width>238</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
border: 2px solid #1E1E1E;
padding-top: -1px;

color: #1E1E1E;
text-align: center;
font-feature-settings: 'clig' off, 'liga' off;
font-family: Montserrat ExtraBold;
font-size: 16px;
font-style: normal;
font-weight: 800;
line-height: 24px;

padding-top: -1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Может быть пустым</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name_4">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>233</y>
      <width>238</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
border: 2px solid #1E1E1E;
padding-top: -1px;

color: #1E1E1E;
text-align: center;
font-feature-settings: 'clig' off, 'liga' off;
font-family: Montserrat ExtraBold;
font-size: 16px;
font-style: normal;
font-weight: 800;
line-height: 24px;

padding-top: -1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Может быть пустым</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name_5">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>292</y>
      <width>238</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
border: 2px solid #1E1E1E;
padding-top: -1px;

color: #1E1E1E;
text-align: center;
font-feature-settings: 'clig' off, 'liga' off;
font-family: Montserrat ExtraBold;
font-size: 16px;
font-style: normal;
font-weight: 800;
line-height: 24px;

padding-top: -1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Может быть пустым</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name_6">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>351</y>
      <width>238</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
border: 2px solid #1E1E1E;
padding-top: -1px;

color: #1E1E1E;
text-align: center;
font-feature-settings: 'clig' off, 'liga' off;
font-family: Montserrat ExtraBold;
font-size: 16px;
font-style: normal;
font-weight: 800;
line-height: 24px;

padding-top: -1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Может быть пустым</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name_7">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>410</y>
      <width>238</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
border: 2px solid #1E1E1E;
padding-top: -1px;

color: #1E1E1E;
text-align: center;
font-feature-settings: 'clig' off, 'liga' off;
font-family: Montserrat ExtraBold;
font-size: 16px;
font-style: normal;
font-weight: 800;
line-height: 24px;

padding-top: -1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Может быть пустым</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company_name_8">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>469</y>
      <width>238</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
border: 2px solid #1E1E1E;
padding-top: -1px;

color: #1E1E1E;
text-align: center;
font-feature-settings: 'clig' off, 'liga' off;
font-family: Montserrat ExtraBold;
font-size: 16px;
font-style: normal;
font-weight: 800;
line-height: 24px;

padding-top: -1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Может быть пустым</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_game_name">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>554</width>
     <height>114</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_game_name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>554</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>Названиe игры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_game_name">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>56</y>
      <width>522</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
border: 2px solid #1E1E1E;
padding-top: -1px;

color: #1E1E1E;
text-align: center;
font-feature-settings: 'clig' off, 'liga' off;
font-family: Montserrat ExtraBold;
font-size: 16px;
font-style: normal;
font-weight: 800;
line-height: 24px;

padding-top: -1px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Игра 999</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_game_level">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>204</y>
     <width>554</width>
     <height>114</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_game_level">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>554</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>Уровень игры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_game_level">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>49</y>
      <width>522</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox#comboBox_game_level{
	border: 2px solid #000;
	border-radius: 8px;
	color: #000;
	font-family: Montserrat ExtraBold;
	font-size: 18px;
	font-style: normal;
	font-weight: 800;
	line-height: 24px; /* 85.714% */
	
}
QComboBox#comboBox_game_level::drop-down{
	border: 0;
	border-radius: 5px;

}

QComboBox#comboBox_game_level::down-arrow{
	image: url();
}</string>
    </property>
    <item>
     <property name="text">
      <string>Цена, Производство, Маркетинг, Кап. Инв., R&amp;D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Цена, Производство, Маркетинг, Кап. Инв</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Цена, Производство, Маркетинг</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Цена, Производство</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Цена</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_game_script">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>328</y>
     <width>554</width>
     <height>179</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_game_script">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>554</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #1E1E1E;
text-align: center;
font-family: Montserrat ExtraBold;
font-size: 24px;
font-style: normal;
font-weight: 800;
line-height: 24px;</string>
    </property>
    <property name="text">
     <string>Сценарий игры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_game_script">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>49</y>
      <width>522</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox#comboBox_game_script{
	border: 2px solid #000;
	border-radius: 8px;
	color: #000;
	font-family: Montserrat ExtraBold;
	font-size: 18px;
	font-style: normal;
	font-weight: 800;
	line-height: 24px; /* 85.714% */
	
}
QComboBox#comboBox_game_script::drop-down{
	border: 0;
	border-radius: 5px;

}

QComboBox#comboBox_game_script::down-arrow{
	image: url();
}</string>
    </property>
    <item>
     <property name="text">
      <string>Стандартный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Рандомный кризис</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_create_new_script">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>114</y>
      <width>522</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

border-radius: 5px;
border: 3px solid rgb(0, 0, 0);
color: rgb(0, 0, 0);
padding-top: -1px;

}

QPushButton::hover{

border-radius: 5px;
border: 3px solid #000;
color:#fff;
padding-top: -1px;
background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #F7931E, stop:1 #EF0009);
}

</string>
    </property>
    <property name="text">
     <string>Создать новый сценарий</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>533</y>
     <width>271</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat ExtraBold</family>
     <pointsize>-1</pointsize>
     <weight>99</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 5px;
border: 3px solid rgba(239, 0, 9, 1);
color: rgb(0, 0, 0);
padding-top: -1px;

font-family: Montserrat ExtraBold;
font-size: 20px;
font-style: normal;
font-weight: 800;
line-height: 24px;
letter-spacing: -1px;
}

QPushButton::hover{
border-radius: 5px;
border: 3px solid rgba(47, 47, 47, 1);
color: #fff;
padding-top: -1px;
background-color:rgba(239, 0, 9, 1);
}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>533</y>
     <width>271</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat ExtraBold</family>
     <pointsize>-1</pointsize>
     <weight>99</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 5px;
border: 3px solid rgba(57, 219, 0, 1);
color: rgb(0, 0, 0);
padding-top: -1px;

font-family: Montserrat ExtraBold;
font-size: 20px;
font-style: normal;
font-weight: 800;
line-height: 24px;
letter-spacing: -1px;
}

QPushButton::hover{
border-radius: 5px;
border: 3px solid rgba(47, 47, 47, 1);
color:#fff;
padding-top: -1px;
background-color:rgba(57, 219, 0, 1);
}

</string>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
